### PR TITLE
added critical value calc for other 2 models

### PR DESCRIPTION
**Pull request by ll01**

### CODE REVIEW

File: src/distrib/dickeyfuller.rs [hunk 1]
```diff
@@ -14,27 +14,75 @@
...
     let (t, u, v, w) = match alpha {
         AlphaLevel::OnePercent => (-3.43035, -6.5393, -16.786, -79.433),
         AlphaLevel::TwoPointFivePercent => (-3.1175, -4.53235, -9.8824, -57.7669),
         AlphaLevel::FivePercent => (-2.86154, -2.86154, -4.234, -40.04),
         AlphaLevel::TenPercent => (-2.56677, -1.5384, -2.809, 0.),
     };
+    return calculate_t_stat_from_estimators(t, u, v, w, sz);
+}
+
+pub fn no_constant_no_trend_critical_value<F: Float>(
```
Comment (ssoudan) [inline src/distrib/dickeyfuller.rs:35]: Please add the docstring.

**Review by ssoudan — CHANGES_REQUESTED** [review]
Thanks! 

Please, add the missing docstrings, run `cargo fmt` on the project, ensure `cargo check`, `cargo test` and `cargo clippy` pass.

File: src/distrib/dickeyfuller.rs (hunk 1 above)
Comment (ll01) [reply src/distrib/dickeyfuller.rs:43]: my bad I didn't format one of the functions properly I just pushed again

Comment (ssoudan) [reply src/distrib/dickeyfuller.rs:43]: thanks

**Review by ssoudan — APPROVED** [review]

**Pull request merged**